src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -229,6 +229,11 @@ private struct MetadataInfoCreator : IDisposable
             // The set of type definitions we've read out of the current metadata reader.
             private readonly List<MetadataDefinition> _allTypeDefinitions;
 
+            // Map from node represents extension method to list of possible parameter type info 
+            // (which can be more than one because we might have multiple overloads).
```

Comment:
how do overloads work in hte UI though...

Reply:
overload isn't the right word here, currently we only return one item if there's multiple overloads for same target type, e.g. 
```cs
public static bool M(this bool x);
public static bool M(this bool x, int y);
```
This here, however, is to support this
```cs
public static bool AnotherExtensionMethod1(this int x);
public static bool AnotherExtensionMethod1(this bool x);
```

Reply:
ah, so we'll have one metadata node for `("AnotherExtensionMethod1", Method)`? basically?

can you doc this example?

Reply:
and what happens if those overloads are in different classes in different namespaces?

Reply:
Done